### PR TITLE
feat: consistently handle bare and prefixed chromosome numbering

### PR DESCRIPTION
**Pull request by jsstevenson**
* Use common processing methods for all cases of chromosome args that transforms bare number strings like `"7"` into UCSC-style prefixed strings like `"chr7"`. 
* Fills in some vagueries surrounding chromosome/accession usage in documentation

### CODE REVIEW

File: src/cool_seq_tool/handlers/seqrepo_access.py [hunk 1]
```diff
@@ -145,12 +145,16 @@ def chromosome_to_acs(self, chromosome: str) -> tuple[list[str] | None, str | No
         acs = []
         for assembly in ["GRCh38", "GRCh37"]:
```
Comment (korikuzma) [inline src/cool_seq_tool/handlers/seqrepo_access.py:146]: Can do this in a separate PR if you want (will have to import Assembly)
```suggestion
        for assembly in Assembly.__members__:
```

Comment (jsstevenson) [reply src/cool_seq_tool/handlers/seqrepo_access.py]: actually, this should be a separate issue, `__members__` gives the keys (`"GRCH38"`) not the values (`"GRCh38"`) and that makes some tests fail

Comment (jsstevenson) [reply src/cool_seq_tool/handlers/seqrepo_access.py:146]: `[v.value for v in Assembly]` but w/e I'm too lazy at this point

Comment (korikuzma) [reply src/cool_seq_tool/handlers/seqrepo_access.py:146]: Haha I can make a new PR for it

**Review by korikuzma — APPROVED** [review]
